cal/cal/static/css/calendaranalytics.scss

```diff
@@ -62,4 +76,4 @@ button {
   }
 }
 
-/*# sourceMappingURL=calendaranalytics.css.map */
\ No newline at end of file
+/*# sourceMappingURL=calendaranalytics.css.map */
```

Comment:
Oh, it might have been from Tiffany. Try merging from master and see if it goes away